include/irr/asset/filters/CSummedAreaTableImageFilter.h

```diff
@@ -375,24 +353,27 @@ class CSummedAreaTableImageFilter : public CMatchedSizeInOutImageFilterCommon, p
 										const size_t scratchOffset = asset::IImage::SBufferCopy::getLocalByteOffset(localCoord, scratchByteStrides);
 										decodeType* entryScratchAdress = reinterpret_cast<decodeType*>(reinterpret_cast<uint8_t*>(scratchMemory) + scratchOffset);
 
-										normalize(entryScratchAdress);
+										if(isSignedFormat)
```

Comment:
signed and normalized format

Reply:
you wamted me to change usual lambda to this afaik but remember we have got a flag that may force normalizing regardless of format

Reply:
I'm concerned about SFLOAT always normalizing inot [1-,1] which is not very helpful.